lib/oktakit/client/authentication.rb

```diff
@@ -0,0 +1,48 @@
+module Oktakit
+  class Client
+    module Authentication
+      # Authenticate User
+      #
+      # @param username [String]. Username for authn request
+      # @param password [String]. Password for authn request
+      # @param mfa [String]. Multi-Factor Token for authn request (only first found "token:software:totp" supported)
+      # @example
+      #   Oktakit.authn('user', 'pass', 123456)
```

Comment:
Is MFA enabled for everyone? Or should the argument be optional? If it's optional, I rather use a keyword argument.